vm/src/stdlib/io.rs

```diff
@@ -701,10 +701,16 @@ mod _io {
     // TextIO Base has no public constructor
     #[pyattr]
     #[pyclass(name = "_TextIOBase", base = "_IOBase")]
+    #[derive(Debug, PyPayload)]
     struct _TextIOBase;
 
     #[pyimpl(flags(BASETYPE))]
-    impl _TextIOBase {}
+    impl _TextIOBase {
+        #[pyattr]
+        fn encoding(ctx: &Context) -> PyObjectRef {
+            ctx.none()
+        }
```

Comment:
```suggestion
        #[pyproperty]
        fn encoding(&self, vm: &VirtualMachine) -> PyObjectRef {
            vm.ctx.none()
        }
```
For adding `encoding` as getset_descriptor, we need to use `#[pyproperty]`

Reply:
When I recomplied your suggestion, I cannot get `None` value. I'm confusing `pyproperty` and `pyattr`.
- python3
```
>>> print(io.TextIOBase().encoding)
None
>>> print(io.TextIOBase.encoding)
<attribute 'encoding' of '_io._TextIOBase' objects>
```
- rustpython
```
>>>>> print(io.TextIOBase().encoding)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: Expected payload '_TextIOBase' but 'TextIOBase' found
>>>>> print(io.TextIOBase.encoding)
<getset_descriptor object at 0x60000341f480>
```

Reply:
`pyproperty` creates a getset field, which is very similar to `property` in python.
`pyattr` creates a simple attribute, same to `self.encoding = None`

Reply:
@Snowapril is it a getset instead of attribute?

Reply:
Yes. It should be `getset`. `getset_repr` will print "<attribute '%V' of '%s' objects>".
Current rustpython getset does not implement `repr`.